src/inference/src/ie_core.cpp

```diff
@@ -1162,6 +1171,10 @@ CNNNetwork Core::ReadNetwork(const std::string& model, const Blob::CPtr& weights
     return _impl->ReadNetwork(model, weights);
 }
 
+ExecutableNetwork Core::LoadNetwork(const CNNNetwork& network, const std::map<std::string, std::string>& config) {
+    return LoadNetwork(network, ov::DEFAULT_DEVICE_NAME, config);
```

Comment:
Why not to use auto explicitly here?

Reply:
we want to distinguish whether user is using the default device name(Auto) or passing the device name "AUTO" within this API.